### DISCUSSION TITLE
How to change week day names and month names in Datepicker?

### DISCUSSION THREAD

**Discussion by mitjarobic** [ant-design/ant-design#44229] [Q&A]
I am using components for moment js:
```
import DatePicker from 'ant-design-vue/es/date-picker/moment';
import TimePicker from 'ant-design-vue/es/time-picker/moment';
```
I am changing the locale as in documentation, but the week days and months are sill in english
```
<template>
  <a-date-picker v-model:value="value" :locale="locale" />
</template>
<script>
  import locale from 'ant-design-vue/es/date-picker/locale/zh_CN';
  import { defineComponent } from 'vue';
  export default defineComponent({
    setup() {
      return {
        locale,
        value: null,
      };
    },
  });
</script>
```
Can somebody please help?

**Comment by SandyInksane** [comment]
I'm facing the same issue. The language pack for date-picker is not providing a list of translated month names.

**Comment by Wxh16144** [comment]
I'm sorry I'm not familiar with vue, you can ask a question at https://github.com/vueComponent/ant-design-vue mirrors, maybe they can solve your problem over there.
